src/posthog-featureflags.ts

```diff
@@ -244,7 +244,14 @@ export class PostHogFeatureFlags {
                 }
                 this.instance.persistence?.register({ [FLAG_CALL_REPORTED]: flagCallReported })
 
-                this.instance.capture('$feature_flag_called', { $feature_flag: key, $feature_flag_response: flagValue })
+                this.instance.capture('$feature_flag_called', {
+                    $feature_flag: key,
+                    $feature_flag_response: flagValue,
+                    $feature_flag_payload: this.getFeatureFlagPayload(key) || null,
+                    $feature_flag_bootstrapped_response: this.instance.config.bootstrap?.featureFlags?.[key] || null,
```

Comment:
Not blocking because I'm asking you to do something a bit more involved, but I think you should probably make this change to the React Native SDK as well, given that we want to have feature parity between the two client-side SDKs as much as we can reasonably can.  [Here's the relevant code](https://github.com/PostHog/posthog-js-lite/blob/main/posthog-core/src/index.ts#L1234-L1260), although I think you might need to change some stuff in `posthog-core` to get it to play nicely.

Reply:
I think that should be here already! https://github.com/PostHog/posthog-js-lite/pull/320

Reply:
🤦 wow now I feel like a jerk for calling it out... nice work man!

Reply:
All good! just updated with $used_bootstrap_value!